cdap-operational-stats-core/pom.xml

```diff
@@ -120,6 +124,14 @@
       <artifactId>jcl-over-slf4j</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
The main reason of avoiding the httpclient library is due to compatibility issue.